lib/schemaUtils.js

```diff
@@ -124,6 +125,22 @@ function hash(input) {
   return crypto.createHash('sha1').update(input).digest('base64');
 }
 
+/**
+ * Remove or leave the deprecated properties according to the option
+ * @param {object} resolvedSchema - the schema to verify properties
+* @param {boolean} includeDeprecated - Whether to include the deprecated properties
+ * @returns {undefined} undefined
+ */
+function verifyDeprecatedProperties(resolvedSchema, includeDeprecated) {
+  traverseUtility(resolvedSchema.properties).forEach(function (property) {
+    if (property) {
```

Comment:
Make sure to check `property` is an object here to avoid type errors with invalid data.

Reply:
Done